RibonDAO/core-api
# feat: add days donating and increment on donation

Adds a new field to user donation stats, and a new job to increment the days donating field on each donation.

## Type of change


## How to test

- Run the AddToDaysDonatingJob
  - Check if it only increments days donating if there wasnt a donation on the same day
